### ISSUE TITLE
Sorting Match replays

### ISSUE THREAD

**Issue by pinheadmercs** [aussieaddons/plugin.video.afl-video#2645]
Hi Andy, 

Firstly I know this is not an error, its more of a request.... so thanks for even taking the time to read this .......... if this is outside any boundaries please delete......

Do you have a way to sort the match replays into the correct sequence, as it stands at the moment, when watch the games through Kodi, they are sorted from Q4 back to Q1. Now I have tried the menu sorting from Ascending to descending and vice versa but to no avail. is this hard set by Telstra/afl or am I just being an eejit and don't know what I am doing.

**Comment by andybotting** [comment]
Yeah this definitely qualifies as an issue. We'll look to implement some method of arranging the list better.

cheers

**Comment by pinheadmercs** [comment]
Magic... many thanks for this and all your time and effort
